src/pages/dashboard/components/HeaderDashboard.tsx

```diff
@@ -350,13 +349,13 @@ const HeaderDashboard = (props: HeaderDashboardProps) => {
             &nbsp;&nbsp;{t("BETA")}
           </Text>
         </Text>
-        <If condition={!isHomepage}>
+        <When condition={!isHomepage}>
           <Text variant="text-16" className="absolute top-3 right-3 text-white">
             {t("Last Updated on {date}", {
               date: isValidDate(lastUpdatedAt) ? new Date(String(lastUpdatedAt))?.toISOString().split("T")[0] : ""
             })}
           </Text>
-        </If>
+        </When>
```

Comment:
:art: in general, I'd like to get rid of this react-if library. It seems to just be syntactic sugar with some annoying performance side effects. Not a big deal to leave this alone, but FYI I've been removing it as I refactor components. This can a simple ternary instead, and then react doesn't have to pretend it's going to render the Text component when it's not:
```
{ 
  isHomepage ? null : <Text ...
  </Text>
}
```